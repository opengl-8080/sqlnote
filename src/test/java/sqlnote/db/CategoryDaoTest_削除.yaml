CATEGORY_NODE:
-
    ID: 1
    TITLE: TEST_A
-
    ID: 2
    TITLE: TEST_B
-
    ID: 3
    TITLE: TEST_C
