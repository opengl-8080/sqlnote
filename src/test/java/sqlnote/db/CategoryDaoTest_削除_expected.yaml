CATEGORY_NODE:
-
    ID: 1
    TITLE: TEST_A
-
    ID: 3
    TITLE: TEST_C
